client-go/cmd/client/local.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Server": "127.0.0.1:8080"
+}
```

Comment:
This should go into a config example directory instead of with the code. Setup could copy it over for easy development use. Also it may be good to make it a template so developers know it's not a final version and the final version is not controlled by source control but explicitly excluded.